examples/xnvme_io_async.c

```diff
@@ -302,11 +302,9 @@ static struct xnvmec_sub g_subs[] = {
 			{XNVMEC_OPT_DATA_OUTPUT, XNVMEC_LOPT},
 			{XNVMEC_OPT_SEED, XNVMEC_LOPT},
 
-			{XNVMEC_OPT_DEV_NSID, XNVMEC_LOPT},
-			{XNVMEC_OPT_BE, XNVMEC_LOPT},
-			{XNVMEC_OPT_ADMIN, XNVMEC_LOPT},
 			{XNVMEC_OPT_ASYNC, XNVMEC_LOPT},
 			{XNVMEC_OPT_DIRECT, XNVMEC_LFLG},
+			XNVMEC_BACKEND_OPTIONS ///< List of opts
```

Comment:
You can remove the ``XNVMEC_OPT_ASYNC`` and ``XNVMEC_OPT_DIRECT`` above ``XNVMEC_BACKEND_OPTIONS``, since ``XNVMEC_BACKEND_OPTIONS`` now adds them.

Reply:
Fixed